blueocean-dashboard/src/main/js/components/IfCapability.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component, PropTypes } from 'react';
+import { capabilityStore } from './Capability';
+
+const { string, object } = PropTypes;
+
+class IfCapability extends Component {
+    render() {
+        const { _class, capability, capabilities } = this.props;
+       
+        if (capabilities[_class].has(capability)) {
+            return this.props.children;
+        }
+
+        return null;
+    }
+}
+
+IfCapability.propTypes = {
+    _class: string,
```

Comment:
I usually reserve prefacing with an underscore for "pseudo private" methods or properties. Perhaps "className" (which you can feel free to repurpose in any React component) or something like "modelClass"? Just a nit.